src/averageAges.js

```diff
@@ -37,7 +41,21 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people
+      .filter(person => person.mother !== null)
```

Comment:
A lot of actions
You can just find women with children or women & calc average)

Reply:
Try to make better